Source/DiabloUI/diabloui.cpp

```diff
@@ -461,6 +462,11 @@ void UiHandleEvents(SDL_Event *event)
 		return;
 	}
 
+	if (event->type == SDL_KEYUP && event->key.keysym.sym == SDLK_PRINTSCREEN) {

```

Comment:
Instead of hard coding against SDLK_PRINTSCREEN we should look at what has been bound to "PRINT" in the keymapper.

Reply:
I have it bound to F9 as Fn+Print Screen is not registered by DevilutionX as PRINT key

Reply:
> Instead of hard coding against SDLK_PRINTSCREEN we should look at what has been bound to "Screenshot" in the keymapper.

My bad for some reason I thought it adjusted what SDLK_PRINTSCREEN is bound to when it initializes the keys.

Reply:
Yeah that explains why it's not working for you